src/language-switcher/class-wpml-elementor-language-switcher-factory.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
@amElnagdy, now that we use namespaces, you can use smaller file names => `factory.php` is enough :)

Reply:
Perfect, thanks @strategio!

As I have you here, to make this whole thing works properly the custom language switcher option in WPML -> Languages has to be enabled and the Dropdown option is selected. 
(As we already using wpml language switcher action)
So, should we enable it programmatically (is it possible?) or ask the clients to do?
What do you think?

Reply:
@amElnagdy, we should enable the custom language switcher programmatically. Please create a ticket in Core and assign it to me. I will provide some method to do this.

Reply:
Created - https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6669